src/operator/tensor/dot-inl.cuh

```diff
@@ -510,6 +398,7 @@ inline void DotCsrDnsDnsImpl(const OpContext& ctx,
     return;
   }
 
+  using namespace mshadow;
   using mshadow::cuda::kBaseThreadNum;
```

Comment:
You may not need the mshadow directive since you added the line above.

Reply:
not used now, deleted.